src/components/listing/ListingPage.jsx

```diff
@@ -1,170 +1,75 @@
-        />
-        <ProductCardVertical
-          item={{
-            productName: "Blue T-Shirt",
-            productShortDescription: "Lorem Ipsum has been the dummy text",
-            productLongDescription: "",
-            trendingText: "New",
-            categoryName: "men",
-            soldBy: "Amazon",
-            rating: 4.4,
-            salePrice: 899,
-            originalPrice: 999,
-            discountPercent: 10,
-            closeActive: false,
-            wishlisted: false,
-            imagePath: "/assets/product.jpg",
-          }}
-        />
-        <ProductCardVertical
-          item={{
-            productName: "Blue T-Shirt",
-            productShortDescription: "Lorem Ipsum has been the dummy text",
-            productLongDescription: "",
-            trendingText: "New",
-            categoryName: "men",
-            soldBy: "Amazon",
-            rating: 4.4,
-            salePrice: 899,
-            originalPrice: 999,
-            discountPercent: 10,
-            closeActive: false,
-            wishlisted: true,
-            imagePath: "/assets/product.jpg",
-          }}
-        />
-        <ProductCardVertical
-          item={{
-            productName: "Blue T-Shirt",
-            productShortDescription: "Lorem Ipsum has been the dummy text",
-            productLongDescription: "",
-            trendingText: "New",
-            categoryName: "men",
-            soldBy: "Amazon",
-            rating: 4.4,
-            salePrice: 899,
-            originalPrice: 999,
-            discountPercent: 10,
-            closeActive: false,
-            wishlisted: false,
-            imagePath: "/assets/product.jpg",
-          }}
-        />
-        <ProductCardVertical
-          item={{
-            productName: "Blue T-Shirt",
-            productShortDescription: "Lorem Ipsum has been the dummy text",
-            productLongDescription: "",
-            trendingText: "New",
-            categoryName: "men",
-            soldBy: "Amazon",
-            rating: 4.4,
-            salePrice: 899,
-            originalPrice: 999,
-            discountPercent: 10,
-            closeActive: false,
-            wishlisted: true,
-            imagePath: "/assets/product.jpg",
-          }}
-        />
-        <ProductCardVertical
-          item={{
-            productName: "Blue T-Shirt",
-            productShortDescription: "Lorem Ipsum has been the dummy text",
-            productLongDescription: "",
-            trendingText: "New",
-            categoryName: "men",
-            soldBy: "Amazon",
-            rating: 4.4,
-            salePrice: 899,
-            originalPrice: 999,
-            discountPercent: 10,
-            closeActive: false,
-            wishlisted: false,
-            imagePath: "/assets/product.jpg",
-          }}
-        />
+        {productsArr.map((item) => {
+          return (
+            <ProductCardVertical
+              key={item._id}
+              item={{
```

Comment:
what is the intention to create object to pass prop?i guess instead you can pass item directly and in destinated component you can destructure whate ever properties of that item you want

Reply:
Will look into this and update in fixes feature